wkhtmltopdf-image/README.md

```diff
@@ -0,0 +1,36 @@
+# WKHtmlToPdf Image
+This docker image definition provides a service wrapper around the command-line open-source wkhtmltopdf generation tool.
+It is based off of the OpenLabs image work [here](https://github.com/openlabs/docker-wkhtmltopdf-aas) with updates
+to include a more recent version of the wkhtmltopdf tool and avoid file-system storage.
+
+This image is designed to be used as a service for generating PDF files from HTML sources.
+
+License information:
+
+Copyright (c) 2014, Openlabs Technologies & Consulting (P) Limited
+All rights reserved.
```

Comment:
Par of using their stuff as a base is that we need to include their copyright and license information.